src/pages/Bookmark/Bookmark.js

```diff
@@ -1,54 +1,76 @@
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
+import axios from 'axios';
+import BottomNav from '../../components/BottomNav/BottomNav';
+import HeaderSearch from '../../components/HeaderSearch/HeaderSearch';
+import Loading from '../../components/Loading/Loading';
 import MainCard from '../../components/MainCard/MainCard';
 import { API } from '../../config';
 import { commonLayOut } from '../../styles/mixin';
 
 const Bookmark = () => {
   const [bookmarkData, setBookMarkData] = useState([]);
-  const {
-    product_id: id,
-    name: title,
-    price,
-    discount,
-    image: thumbnail,
-    new: isNew,
-    hot: isHot,
-    rating: grade,
-  } = bookmarkData;
+  const [isLoading, SetIsLoading] = useState(true);
 
   useEffect(() => {
-    fetch(`${API}/orders?page=1offset=0&limit=4`)
-      .then(res => res.json())
-      .then(res => setBookMarkData(res));
+    const authToken = localStorage.getItem('Token');
+    try {
+      axios(`${API}/orders/bookmark?offset=0&limit=4`, {
+        headers: {
+          Authorization: authToken,
+        },
+      }).then(res => setBookMarkData(res.data));
+    } catch (error) {
+      console.error(error);
+    }
+    SetIsLoading(false);
   }, []);
 
-  return (
-    <BookMarkWrapper>
-      <CardListWrapper>
-        <MainCard
-          id={id}
-          title={title}
-          price={price}
-          listPrice={parseInt(price) * (1 - discount)}
-          thumbnail={thumbnail}
-          isNew={isNew}
-          isHot={isHot}
-          grade={grade}
-        />
-      </CardListWrapper>
-    </BookMarkWrapper>
+  return isLoading ? (
+    <Loading />
+  ) : (
```

Comment:
이렇게 써도 되지만 if문으로 나누면 좀 더 깔끔하게 쓸 수 있습니다.
```suggestion
if(isLoading) return <Loading />

return ( <BookMarkWrapper> .
...
```

Reply:
수정했습니다😄